test/internet/test-dgram-broadcast-multi-process.js

```diff
@@ -28,14 +28,14 @@ const assert = require('assert');
 const dgram = require('dgram');
 const util = require('util');
 const networkInterfaces = require('os').networkInterfaces();
-const fork = require('child_process').fork;
+const { fork } = require('child_process');
 const LOCAL_BROADCAST_HOST = '255.255.255.255';
 const TIMEOUT = common.platformTimeout(5000);
 const messages = [
   Buffer.from('First message to send'),
   Buffer.from('Second message to send'),
   Buffer.from('Third message to send'),
-  Buffer.from('Fourth message to send')
+  Buffer.from('Fourth message to send'),
```

Comment:
why add a comma

Reply:
From https://eslint.org/docs/rules/comma-dangle:

> Trailing commas simplify adding and removing items to objects and arrays, since only the lines you are modifying must be touched. Another argument in favor of trailing commas is that it improves the clarity of diffs when an item is added or removed from an object or array:
<img width="211" alt="Screen Shot 2019-03-21 at 8 30 39 PM" src="https://user-images.githubusercontent.com/718899/54798953-3c884380-4c18-11e9-9d4c-d39a737b2f35.png">

Reply:
tks